snippets/csharp/System.Linq/IOrderedEnumerableTElement/Overview/Project.csproj

```diff
@@ -1,8 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
```

Comment:
Curious why these project files are being deleted/renamed. If the projects are being restructured, what's the end goal here?

Reply:
the MS process mandates that there are .csproj files so that CI can validate. All good.
However it makes sense to knit *.csproj into a VS2022 .sln file so that all the analyzer improves can be applied together.
Having multiple x.csproj files (e.g. program.csproj) is not acceptable to VS2022 so I renamed to _unique_.csproj

I agree with Eirik's other points of about top-level statements, NS (don't know how I missed those!)
But I lean towards analyzer recommendations [e.g. prefer .Count>0 or .Length>0 rather than Any() for perf]
having a splurge of Warnings/Messages is untidy and imho bad UX.

I was not trying to sneak in contrary .editoconfig settings for long-term, but I despise "private" accessors as this is default and imho just contributes unnecessary noise to detriment of the code, i.e. I pick
dotnet_style_require_accessibility_modifiers = omit_if_default
but once I have cleaned-up such noise I am happy to revert to owner's version ["private" shouldn't re-appear]

I set the "let owner/admin modify" [sorry I can't remember exact phrase], so by all means give the changes another spin
I will watch reviewer comments with interest!

I appreciate the effort to update the doc snippets to portray latest C# semantics is a bore, but we all want such docs to give good guidance. Sometimes some ugly example throws me into digging up all the rabbit-holes [OCD!]

Anyway I am pleased the PR is now being looked at as otherwise my wasted effort crafting it!